tests/test_feather_eager.py

```diff
@@ -37,9 +38,7 @@ def test_feather_format():
     }
     df = pd.DataFrame(data).sort_index(axis=1)
     with tempfile.NamedTemporaryFile(delete=False) as f:
-        df.to_feather(f)
-
-    df = pd.read_feather(f.name)
+        pa.feather.write_feather(df, f, version=1)
```

Comment:
version 2 is the includes compression by default, is only version 1 working when reading reading back in?

Reply:
Thanks @BryanCutler for the review. It only works for version 1 now as version 2 changed the magic string from `FEA1` to `ARROW1` (and a few other changes in place). I am thinking of adding version 2 support in follow up PRs. This PR is just to bring arrow library to the latest version.

Reply:
Ok great, SGTM